sonar/theme/templates/sonar/frontpage.html

```diff
@@ -20,11 +20,13 @@
 
 {%- extends config.BASE_TEMPLATE %}
 
+{%- set body_css_classes=['homepage'] %}
+
 {% block header %}
 <div class="bg-organisation p-4">
   <div class="container">
     <div class="row justify-content-center">
-      <div class="col-8 col-xs-6 col-lg-3 py-3 py-sm-5 text-center">
+      <div class="col-8 col-xs-6 col-lg-3 py-3 text-center">
```

Comment:
`col-8` ? seems strange with other col-* isn't it ?

Reply:
No, I think it this is what we really want.